include/swift/AST/ModuleDependencies.h

```diff
@@ -35,6 +36,7 @@ class Identifier;
 
 /// Which kind of module dependencies we are looking for.
 enum class ModuleDependenciesKind : int8_t {
+  FirstKind,
   SwiftTextual,
```

Comment:
Thanks! Please also explicitly add `SwiftTextual = FirstKind` and `LastKind = Clang + 1` in the definition.